README.md

```diff
@@ -87,7 +99,7 @@ There are two ways to authenticate:
 > **Warning**
 > Authenticating as a user is not supported for **private cloud** tenants. Instead, those users should authenticate with client credentials.
 
-## Available Commands
+## 💻 Available Commands
 
 - [auth0 actions](https://auth0.github.io/auth0-cli/auth0_actions.html) - Manage resources for actions
 - [auth0 api](https://auth0.github.io/auth0-cli/auth0_api.html) - Makes an authenticated HTTP request to the Auth0 Management API
```

Comment:
`auth0 ips` shouldn't be in the top-level command list anymore, as it now nested under another command (L111).

Reply:
Good catch 👍🏻 [`3549bc7` (#622)](https://github.com/auth0/auth0-cli/pull/622/commits/3549bc787d1980d30d1feb1fed50394cb8d5b657)